usaspending_api/references/test_agency_loader.py

```diff
@@ -22,13 +22,13 @@ def test_department(self):
         Make sure an instance of a department is properly created
         """
 
-        department = Agency.objects.get(cgac_code='002', fpds_code='0000', subtier_code='0000')
+        department = Agency.objects.get(toptier_agency__cgac_code='002', toptier_agency__fpds_code='0000', subtier_agency__subtier_code='0000')
```

Comment:
Forgive me, I'm still grokking the agencies...are FPDS codes the same thing as CGAC codes? If so, why do we always to lookups on fpds codes? That's legacy info, yes?